torch/csrc/jit/script/interfaces.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/csrc/jit/script/module.h>
+
+namespace torch {
+namespace jit {
+namespace script {
+
+TORCH_API Value* tryMatchInterfaceOps(
```

Comment:
I really do not think this is a good idea. The entire point of having schema is so that custom rules for matching operators are never written. This interface opens up a HUGE loophole that will just be a dumping group for special snowflakes.

Simple solution for now: just fail dynamically if the type doesn't have a `__lt__` method. It's not like we are benefiting from this static information to begin with.